classes/book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './classes/item.rb'
+class Book < Item
+  attr_accessor: :publisher, :cover_state
```

Comment:
attr_accessor: should be attr_accessor

Reply:
Sorry I don't fully understand. Is there something wrong with the way it was spelt?